protos/services.proto

```diff
@@ -32,13 +33,24 @@ message GetProcessMemoryResponse {
   bytes memory = 1;
 }
 
+message GetSymbolsRequest {
+  uint32 process_id = 1;
```

Comment:
Sorry - github lets you select multiple lines but does show it clearly... What I meant in previous comment is to remove process_id as well. The file is not tied to a process. It is just a file on the disk.

Reply:
see other comment